README.md

```diff
@@ -136,145 +124,167 @@ Hvis en oppgave bare skal vises i sin spilleliste, kan `indexed: false` legges
+```
 
+![](imgs/tips.png)
 
-#### Scratch kodeblokker
+Eller ting som kan prøves ut:
 
-Scratchkode kan skrives rett inn i Markdown-teksten. Denne blir oversatt til
-figurer av et verktøy som heter [Scratchblocks2][sb2]. På hjemmesidene til
-Scratch finnes [dokumentasjon over syntaks][doc sb]. Et nyttig verktøy er
-[scratchblocks translator][sbt] som lar deg hente ut og oversette scratchkode.
+```
+## Ting å prøve {.try}
+Klarer du å fly helt til månen?
+```
 
-[sb2]: https://github.com/blob8108/scratchblocks2 "Scratchblocks2"
-[doc sb]: http://wiki.scratch.mit.edu/wiki/Block_Plugin/Syntax "Dokumentasjon scratchblocks"
-[sbt]: http://scratchblocks.codeclub.org.uk/translator/ "Scratchblocks translator"
+![](imgs/prov.png)
 
-Hele kodesnutter skrives som et eget avsnitt på følgende måte:
+Eller test prosjektet:
 
-<pre>
-```blocks
-når grønt flagg trykkes
-for alltid
-pek mot [musepekeren v]
-slutt
 ```
-</pre>
+## Test prosjektet {.flag}
+Nå er det smart å teste at programmet fungerer som det skal.
+- Stopper programmet når du kræsjer i veggen?
+- Får du poeng når du hopper over hinderne?
+```
 
-Rykkes koden ekstra inn i lister, vil den flyte med teksten rundt. For eksempel
-vil man typisk ha innrykk på fire mellomrom for kode som er en del av en
-sjekkliste:
+![](imgs/test_prosjektet.png)
 
-<pre>
-## Sjekkliste {.check}
-- En forklarende tekst her. Og kode som hører til:
+Eller lagre:
 
-    ```blocks
-    for alltid
-    pek mot [musepekeren v]
-    ```
+```
+## Lagre prosjektet {.save}
+Ikke glem å lagre prosjektet. Du kan kalle filen `skilpadde.py`. **Ikke** lagre
+som `turtle.py`, da blir python forvirret på kodelinjen `from turtle import *`.
+```
 
-- Neste sjekkpunkt.
-</pre>
+![](imgs/lagre_prosjektet.png)
+
+
+#### Nivå og PR
+Gjør deg noen tanker på **nivået** (`level`) til oppgaven når du skriver
+oppgaven. Her er noen beskrivelser som kan hjelpe.
+
+- `level: 1` **introduksjonsoppgave**: Alle uten programmeringskunnskap burde ha
+  mulighet til å følge oppgaven. Oppgaven burde være selvstendig, slik at man
+  ikke trenger å sjekke opp andre ressurser for tilleggsinformasjon. Oppgaven
+  kan gjerne ha kodeblokker som det ikke er meningen at eleven skal forstå, men
+  som viser hva som er mulig i programmeringsspråket. Kodeblokker kan gjerne
+  være fullstendige, slik at kopier/lim-inn er tilstrekkelig for å komme seg
+  gjennom oppgaven.
+- `level: 2` **nybegynner**: Det anntas at eleven kan bruke editoren og vet
+  hvordan koden kjøres. Bygg opp meget enkle utfordringer med nesten identiske
+  eksempler, slik at eleven ikke setter seg fast, samt at man unngår passiv
+  kopier/lim-inn.
+- `level: 3` **erfaren**: Eleven kan lage program og kjøre de. Eleven kan løse
+  enkle problem på egen hånd. Hjelp eleven å løse middels vanskelige problem
+  med lignende eksempler. Link gjerne til eksterne ressurser som API i
+  `## Utfordring {.challange}`, slik at eleven blir flink til å utforske.
```

Comment:
Ble noen skriveleif. Har kjørt endel `--force`, for håpe du ikke ble berørt :grin: